### DISCUSSION TITLE
[Transmogification] Minor UI Tweaks

### DISCUSSION THREAD

**Discussion by GreenhornCode** [runelite/runelite#15077] [Ideas]
While trying on several different items, I found two small inconveniences that I felt could be adjusted. 

First, I would prefer the search results for items to be sorted alphabetically. When I'm not looking for any specific item, I'm just going down the list trying on items and seeing if I like them, or if they fit with the ones I've already selected. Alphabetical order would just make the process a bit smoother in my opinion. In the same vein, I found it a bit tedious to click the equipment slot each time after trying on an item and deciding I didn't like it. I propose to simply remove the statements closing the search box when an item is selected. There are still the options of the escape key and the exit button to close the box.

I've already made these changes for myself and would be glad to contribute them if they are desirable features.

**Comment by Nightfirecat** [comment]
Issues and suggestions for hub plugins must be made on that plugin's repository; they are not maintained by us. The repo is usually accessible from the Support link on the plugin.

**Reply by GreenhornCode** [reply]
My apologies.
